bootstrap.d/gui-apps.yml

```diff
@@ -45,3 +45,45 @@ packages:
       - args: ['ninja', 'install']
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
+
+  - name: wlgears
+    architecture: '@OPTION:arch@'
+    metadata:
+      summary: A port of glxgears to wayland
+      description: "This utility is a port of the glxgears demo of mesa-demos onto wayland with a meson build system."
+      spdx: MIT
+      website: 'https://github.com/no92/wlgears'
+      maintainer: "no92 <no92.mail@gmail.com>"
+      categories: ['gui-apps']
+    source:
+      subdir: ports
+      git: 'https://github.com/no92/wlgears.git'
+      branch: master
```

Comment:
Ideally we pin a commit, which can be overlooked because you control the repo, but please do pin a commit.
You're also lacking a version tag, which is required.

Reply:
Fixed.